client/package.json

```diff
@@ -45,5 +45,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
```

Comment:
How come this is added again in the PR? Does this happen whenever you are running `yarn install`?